### PR TITLE
Try a little harder to find a good level 0 thumbnail image

### PR DESCRIPTION
**Pull request by cbeer**

### CODE REVIEW

File: src/lib/ThumbnailFactory.js [hunk 1]
```diff
@@ -80,16 +86,62 @@ class ThumbnailFactory {
...
-    if (isLevel0ImageProfile(service)) {
-      return ThumbnailFactory.staticImageUrl(resource);
-    }
-
     const aspectRatio = resource.getWidth()
       && resource.getHeight()
       && (resource.getWidth() / resource.getHeight());
 
-    if (maxHeight && maxWidth) {
+    // just bail to a static image, even though sizes might provide something better
```
Comment (mejackreed) [inline src/lib/ThumbnailFactory.js:93]: Is this handling the case where an image information response provides a size but the image server does not respond with it?

Comment (cbeer) [reply src/lib/ThumbnailFactory.js:93]: if the manifest is lying about the available sizes?

Comment (mejackreed) [reply src/lib/ThumbnailFactory.js:93]: Yes

Comment (cbeer) [reply src/lib/ThumbnailFactory.js:93]: Nope; I don't think we handle that case for any of our thumbnail calculations, before or after this PR. Do we have examples of that in the wild?

Comment (mejackreed) [reply src/lib/ThumbnailFactory.js:93]: No I was just trying to better understand what was happening here. But I think I get it now

**Pull request merged**